app/javascript/app/components/metadata-text/metadata-text-component.jsx

```diff
@@ -50,110 +50,46 @@ class MetadataText extends PureComponent {
   // eslint-disable-line react/prefer-stateless-function
   render() {
     const { data, className, showAll } = this.props;
-    const {
-      learn_more_link,
-      source_organization,
-      technical_title,
-      summary,
-      citation,
-      cautions,
-      geographic_coverage,
-      description,
-      date_of_content,
-      summary_of_licenses,
-      terms_of_service_link
-    } = this.props.data;
+    const { title, learnMore, sourceOrganization, citation } = this.props.data;
```

Comment:
I made some changes in this commit that might be worth you having a quick look at on monday, @Bluesmile82 . Just in case! =D